src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -3883,6 +3851,9 @@ public static bool ConnectAsync(SocketType socketType, ProtocolType protocolType
             return pending;
         }
 
+        /// <summary>Binds an unbound socket to "any" if necessary to support a connect.</summary>
+        partial void WildcardBindForConnectIfNecessary(AddressFamily addressFamily);
```

Comment:
nit: should this method be specific about using 'private'?  I noticed it doesn't have any private/internal/public modifier? CoreFx guidelines mention that it should be explicitly specified.

Can both 'partial' and 'private' be specified together?

Reply:
Unfortunately partial methods are implicitly private and prohibit specifying it explicitly:
```
CS0750: A partial method cannot have access modifiers or the virtual, abstract, override, new, sealed, or extern modifiers
```
I don't know why. @madstorgersen?